runtime/battery-station/src/lib.rs

```diff
@@ -101,32 +112,92 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     state_version: 1,
 };
 
+#[derive(scale_info::TypeInfo)]
+pub struct ContractsCallfilter;
+
+// Currently disables Court, Rikiddo and creation of markets using Court or SimpleDisputes
+// dispute mechanism.
+impl Contains<RuntimeCall> for ContractsCallfilter {
+    fn contains(runtime_call: &RuntimeCall) -> bool {
+        #[allow(clippy::match_like_matches_macro)]
+        match runtime_call {
+            RuntimeCall::AssetManager(transfer { .. }) => true,
```

Comment:
What about calls into `xtokens`?

Reply:
I don't see any use for `xtokens`, as it only allows the smart contract to transfer tokens to another chain, but then it consequently loses control of them. In my current understanding, this only open ups gateways to potential abuse without not much chances to provide any value by having those.
On the contrary, calling into `AssetManager` allows the contract to send tokens (besides ZTG which the contract can already transfer) on behalf of the user into the contract, where it then can use those to provide useful applications such as an order book for outcome tokens etc.